build.cake

```diff
@@ -243,6 +243,8 @@ private void DoPackage(string project, string framework, string version, string
 
 private void SignAndTimestampBinaries(string outputDirectory)
 {
+    if (BuildSystem.IsLocalBuild) return;
```

Comment:
When working on the xplat pitch I ran **a lot** of 🍰 builds locally, when building locally signing isn't _really_ necessary and it could take up to 3-4 minutes to sign all the binaries as we build for many, many different runtimes so disabling it locally means quicker turn around when doing local development.

Reply:
You may want to add a comment to the code explaining the why